modules/nw-metalLB-upgrading-operator.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
```

Comment:
Is this file still rendered anywhere? If so we need to change it. It still contains the patching of operatorgroup.